third_party/envoy-gateway/config-gateway.yaml

```diff
@@ -25,14 +25,14 @@ data:
   external-gateways: |
     - class: eg-external
       gateway: eg-external/eg-external
-      service: envoy-gateway-system/knative-external
+      service: eg-external/knative-external
```

Comment:
this is interesting - did they change where services are created?

Reply:
Though if we do that change then it might break upgrades which makes everything more complex.

Actually checking now we don't release the envoy yamls as part of the release so maybe we're ok in terms of what backwards compatibility we support.

Reply:
Can we still drop helm and just apply the single yaml file :P

Reply:
@dprotaso 
I've completed it. Cloud you review again? 🙏